test/e2e/e2e_suite_test.go

```diff
@@ -149,6 +154,50 @@ func (acp *AzureClusterProxy) CollectWorkloadClusterLogs(ctx context.Context, na
 			}(pod, container)
 		}
 	}
+
+	Byf("Dumping workload cluster %s/%s Azure activity log", namespace, name)
```

Comment:
I'll probably refactor this into its own func

Reply:
The kube-system logger and the events should both be funcs. I should have extracted the func in my previous PR.